restart-helpers.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+helm uninstall h1
```

Comment:
a comment on what these commands do would be useful. `install` feels like it could take some time (e.g., does it include building a binary?)

Reply:
It's very quick, like 5 secs. I'll add a note